src/engine/Loader.ts

```diff
@@ -7,6 +7,10 @@ import { ILoadable } from './Interfaces/ILoadable';
 import { ILoader } from './Interfaces/ILoader';
 import { Class } from './Class';
 import * as DrawUtil from './Util/DrawUtil';
+import { obsolete } from './Util/Decorators';
+
+const loaderCss = require('./Loader.css').toString();
```

Comment:
Why require? If TS complains you just need a files.d.ts that declares valid extensions to import

Reply:
This is documented in the Handbook but I can help too if needed

Reply:
Haha I'm used to reacing for `require`, good to know I'll update this